go/vt/external/golib/sqlutils/sqlutils_test.go

```diff
@@ -0,0 +1,257 @@
+/*
+   Copyright 2024 The Vitess Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package sqlutils
+
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type any = interface{}
+
```

Comment:
Why not dropping this type alias and directly using the Golang builtin `any`?

Reply:
the builtin does the exact same thing:
```go
// any is an alias for interface{} and is equivalent to interface{} in all ways.
type any = interface{}
```

Reply:
Ahh, didn't know that was a thing in go

Reply:
fixed